dist-test/docker-compose.yml

```diff
@@ -52,7 +52,7 @@ services:
       - ${FAASM_LOCAL_DIR}:/usr/local/faasm
     working_dir: /build/faasm
     environment:
-      - LOG_LEVEL=info
+      - LOG_LEVEL=debug
```

Comment:
Sorry for necroposting, just updating on last week's changes. Are we sure we want this set to `debug`?

Reply:
Yes, for now given that there are only three of them and they fail so often we can stick with debug